docs/claims.md

```diff
@@ -0,0 +1,175 @@
+
+Now a green box will appear telling you the amount to claim with a "Claim" button to make the claim.
+Click on the "Claim" button and click "Submit (no signature)" to complete the claim.
+
+![claim-9](assets/new-claims/claim-9.png)
+
+At this point you will see a success message if everything went right and your DOTs will now be in
+the account that you claimed to.
+
+![claim-10](assets/new-claims/claim-10.png)
+
+Congratulations you can now participate in aspects of the Polkadot network such as
+[governance](learn-governance) and [staking](learn-staking). During the soft launch period balance
+transfers will not be enabled.
+
+#### Verifying your Claim
+
+After you make an on-chain claim for DOTs, your balance should be updated on the Polkadot UI
+immediately.
+
+Having trouble? Get support in the DOT [Claims Support][] channel.
+
+### Third Party Claims Processes
+
+\_COINBASE_TO_DO
+
+**We do not recommend using a third-party app or process to perform your claim or acquire DOT**
+
+Claiming using a third-party process can lead to the loss of your allocation, therefore we cannot
+recommend using any third party apps to do so. Manually specifying your transaction data, as
+specified in our claims process, is the only way to be certain you will receive your allocation.
+
+## Attesting to a statement
+
+If you've already made a pre-genesis claim, you still have to attest to a statement using your
+Polkadot account.
+
+### What you will need
+
+- Your Polkadot account unlocked on Polkadot-JS Apps UI.
+
+You will be sending a free transaction from your Polkadot account on the Polkadot-JS Apps UI. Once
+you make this transaction you will have the tokens available in your account to use for staking,
+voting, and eventually transfers.
+
+### Log on to Polkadot-JS Apps
+
+Proceed to [polkadot-js Apps][claims app]. You will need to grant Apps access to your account in
+some way. One way would be to go to the Accounts page and "create" a new account, replacing the
+generated seed or mnemonic with the one belonging to your account. The other way is by using the
+Polkadot-JS extension and entering your seed or mnemonic there, which is generally safer than
+entering it directly to the Apps page.
+
+### Make the attestation
+
+Once you have your account entered you should see a red counter appear on the "Claim Tokens" tab on
+the left navigation drawer.
+
+![claim-attest-1](assets/new-claims/claim-attest-1.png)
+
+Click on the "Claim Tokens" tab and you will see a large notification at the top of the page that
+tells you that you need to sign an attestation.
+
+![claim-attest-2](assets/new-claims/claim-attest-2.png)
+
+You will select your Polkadot account that is being notified to you needs to make an attestation in
+the selector on this page. Click "Continue" and you should see a green box appear to the right of
+the selector.
+
+![claim-attest-3](assets/new-claims/claim-attest-3.png)
+
+![claim-attest-4](assets/new-claims/claim-attest-4.png)
```

Comment:
Same here

<img width="606" alt="Screenshot 2020-05-25 at 11 27 07" src="https://user-images.githubusercontent.com/1293565/82799703-bea67480-9e7a-11ea-89bb-f3f41bdfc234.png">

Reply:
Would it be possible to change "Attestation for:" to "Account balance:"?  Might be confusing otherwise, people think they need to pay that amount.